apps/explorer/lib/pages/transaction-details/transaction-card.tsx

```diff
@@ -30,6 +31,9 @@ export const TransactionCard = ({ txHash }: TransactionCardProps) => {
   }
 
   const transactionFee = (Number(tx.gasUsed) * Number(tx.gasPrice)) / 1e18;
+  const confirmations = tx.accessList
+    ?.map((item) => item?.address)
+    .filter(Boolean) as string[];
```

Comment:
suggestion: The `as string[]` assertion bypasses type checking, so can we use `type-guard` here.

```suggestion
      .filter((addr): addr is string => Boolean(addr))) ?? [];
```